.github/workflows/ci.yml

```diff
@@ -0,0 +1,118 @@
+            make_command: make package
+          - os: ubuntu-18.04
+            name: Ubuntu Nightly
+            channel: nightly
+            build_command: cargo check
+            install_deps_command: |
+                                  sudo apt install -f -y cmake
+                                  GLFW_DIR="$GITHUB_WORKSPACE/libglfw3"
+                                  mkdir $GLFW_DIR
+                                  curl -o $GLFW_DIR/libglfw3-dev_3.3.2-1_amd64.deb http://security.ubuntu.com/ubuntu/pool/universe/g/glfw3/libglfw3-dev_3.3.2-1_amd64.deb
+                                  curl -o $GLFW_DIR/libglfw3_3.3.2-1_amd64.deb http://security.ubuntu.com/ubuntu/pool/universe/g/glfw3/libglfw3_3.3.2-1_amd64.deb
+                                  sudo apt install -f -y $GLFW_DIR/libglfw3_3.3.2-1_amd64.deb
+                                  sudo apt install -f -y $GLFW_DIR/libglfw3-dev_3.3.2-1_amd64.deb
+                                  rm -rf $GLFW_DIR
+            make_command: make VERBOSE=1
+          - os: windows-2019
+            name: Windows Stable
+            channel: stable
+            build_command: rustup default stable-msvc; cargo check
+            make_command: make package
+          - os: windows-2019
+            name: Windows Nightly
+            channel: nightly
+            build_command: rustup default nightly-msvc; cargo check
+            install_deps_command: |
+                                  # choco exit with code 1 after successful install ¯\_(ツ)_/¯
+                                  choco install -y --force cmake make | exit 0
+
+                                  curl -sL -o glfw.zip https://github.com/glfw/glfw/archive/3.3.zip
+                                  unzip -qq glfw.zip -d $GITHUB_WORKSPACE
+                                  cd $GITHUB_WORKSPACE/glfw-3.3/
+                                  export GLFW3_INCLUDE_DIR=`pwd`/include
+                                  export GLFW3_INSTALL_DIR=`pwd`/install
+                                  cmake . -DCMAKE_INSTALL_PREFIX=$GLFW3_INSTALL_DIR -DCMAKE_GENERATOR_PLATFORM=x64
+                                  cmake --build . --target install
+            make_command: CMAKE_PREFIX_PATH=$GITHUB_WORKSPACE/glfw-3.3/install make VERBOSE=1
+    steps:
+    - uses: actions/checkout@v2
+    - if: matrix.channel == 'nightly'
+      name: Install latest nightly
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: nightly
+        override: true
+    - if: contains(matrix.build_command, 'clippy')
+      run: rustup component add clippy
+    - name: Install dependencies
+      run: ${{ matrix.install_deps_command }}
+      shell: bash
+    - name: cargo test
+      run: cargo test
+    - if: contains(matrix.build_command, 'clippy')
+      name: cargo clippy
+      run: ${{ matrix.build_command }}
+    - name: MacOS vulkan-portability
+      if: contains(matrix.os, 'macos') && !contains(matrix.build_command, 'ios')
```

Comment:
@kvark Should we test ios with vulkan portability?

Currently it doen't compile. But i can send PR's to fix this.

Reply:
Ideally, yes. Thank you!

Reply:
Wait, wait, no. Vulkan Portability in wgpu is only for debug purposes.
No need to have it on iOS, we have Metal for that!